UPGRADE-4.3.md

```diff
@@ -77,7 +77,7 @@ Form
 FrameworkBundle
 ---------------
 
- * Deprecated the `framework.templating` option, use Twig instead.
+ * Deprecated the `framework.templating` option, configure the twig bundle instead. If twig is already configured, you can safely remove this option.
```

Comment:
`safely remove`? is that actually correct? I mean I still need to fix my code if I rely on `EngineInterface` somewhere, right?

Reply:
I'm not sure how the configuration for this works, but seems like removing it would trigger an error you'd have to fix in that case.

Reply:
Uppercase Twig + maybe remove the second sentence?

Reply:
Reading it again, it actually says the same twice, removed the second part